src/components/Todo/TodoNavigator.tsx

```diff
@@ -1,31 +1,41 @@
 import React from 'react';
 import {createStackNavigator} from '@react-navigation/stack';
 import Todo from './Todo';
-import {Image} from 'react-native';
+import {Image, View, Text} from 'react-native';
+import {useNetInfo} from '@react-native-community/netinfo';
+import styles from './style';
 
 export type TodoStackParamsList = {
   Todo: undefined;
 };
 
 const Stack = createStackNavigator<TodoStackParamsList>();
 
-const LogoTitle = () => (
-  <Image
-    source={{
-      uri:
-        'https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/090/thumb/EGH_ReactNative_ToDo-Final.png',
-      width: 40,
-      height: 40,
-    }}
-  />
-);
+const LogoTitle = () => {
+  const {isInternetReachable} = useNetInfo();
+  return (
+    <View style={styles.logoTitle}>
+      <Image
+        source={{
+          uri:
+            'https://d2eip9sf3oo6c2.cloudfront.net/series/square_covers/000/000/090/thumb/EGH_ReactNative_ToDo-Final.png',
```

Comment:
Unless its dynamic images, recommend to use local assets. 
download and keep in assets folder.